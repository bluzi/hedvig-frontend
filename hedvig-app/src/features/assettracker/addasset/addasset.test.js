import React from "react"
import { shallow } from "enzyme"
import toJson from "enzyme-to-json"

import { AddAssetComponent as AddAsset } from "."

describe('<AddAsset />', () => {
  it('Should render correctly', () => {
    const component = shallow(<AddAsset />)

    expect(toJson(component)).toMatchSnapshot()
  })
})
