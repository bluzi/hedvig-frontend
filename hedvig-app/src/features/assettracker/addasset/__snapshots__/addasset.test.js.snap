// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddAsset /> Should render correctly 1`] = `
<KeyboardAvoidingView
  behavior="padding"
  keyboardVerticalOffset={32}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "justifyContent": "flex-start",
    }
  }
>
  <NavBar
    headerLeft={<Styled(View) />}
    headerRight={<Styled(View) />}
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
        "paddingBottom": 16,
        "paddingTop": 16,
      }
    }
  >
    <TouchableOpacity
      activeOpacity={0.2}
      onPress={[Function]}
    >
      <Image
        source={1}
        style={
          Object {
            "height": 56,
            "width": 56,
          }
        }
      />
    </TouchableOpacity>
  </View>
  <View
    style={
      Object {
        "flex": 4,
        "justifyContent": "flex-start",
      }
    }
  >
    <View
      style={
        Object {
          "borderBottomColor": "#F1F3F7",
          "borderBottomWidth": 1,
          "height": 64,
          "justifyContent": "center",
        }
      }
    >
      <TextInput
        allowFontScaling={true}
        placeholder="Vad är det för pryl?"
        style={
          Object {
            "alignItems": "center",
            "fontFamily": "merriweather",
            "fontSize": 18,
            "height": 40,
            "marginHorizontal": 24,
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <View
      style={
        Object {
          "borderBottomColor": "#F1F3F7",
          "borderBottomWidth": 1,
          "height": 64,
          "justifyContent": "center",
        }
      }
    >
      <TextInput
        allowFontScaling={true}
        placeholder="Vad köpte du den för?"
        style={
          Object {
            "alignItems": "center",
            "fontFamily": "merriweather",
            "fontSize": 18,
            "height": 40,
            "marginHorizontal": 24,
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
    <TouchableOpacity
      activeOpacity={0.2}
      onPress={[Function]}
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "backgroundColor": "white",
          "borderColor": "#651EFF",
          "borderRadius": 24,
          "borderWidth": 1,
          "justifyContent": "center",
          "marginTop": 16,
          "paddingHorizontal": 16,
          "paddingVertical": 8,
        }
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#651EFF",
            "fontFamily": "circular",
            "fontSize": 16,
          }
        }
      >
        Spara
      </Text>
    </TouchableOpacity>
  </View>
</KeyboardAvoidingView>
`;
